bot/kodiak/queries.py

```diff
@@ -792,7 +792,7 @@ async def get_default_branch_name(self) -> Optional[str]:
         data = res.get("data")
         errors = res.get("errors")
         if errors is not None or data is None:
-            logger.error("could not fetch default branch name", res=res)
+            self.log.error("could not fetch default branch name", res=res)
```

Comment:
we configure this in the init method, should just work:

https://github.com/chdsbd/kodiak/blob/2ecca4ed181432b192292084719e3796d9997812/bot/kodiak/queries.py#L746-L748